gapis/api/transform2/transform_test.go

```diff
@@ -112,6 +112,14 @@ func (t *multiplier_transform) RequiresAccurateState() bool {
 	return false
 }
 
+func (t *multiplier_transform) RequiresInnerStateMutation() bool {
+	return false
+}
+
+func (t *multiplier_transform) SetInnerStateMutationFunction(stateMutator transform2.StateMutator) {
```

Comment:
It's a shame that every transform needs to implement this function. I guess go style polymorphism makes it hard to avoid this.

Reply:
Actually can we just pass this in to TransformCommands as an argument. The transforms do not need to store it locally since they will only call it inside that one function. So it could just be provided from the outside to avoid all these trivial functions.

Reply:
Let's talk about this in detail later and if we decide it can go to another PR.